src/modules/add-site/components/blueprints.tsx

```diff
@@ -267,40 +395,63 @@ export function AddSiteBlueprintSelector( {
 				{ __( 'Start from a blueprint' ) }
 			</Heading>
 
+			<BlueprintDetailsModal
+				warnings={ blueprintWarnings }
+				validationError={ validationError }
+				fileName={ selectedFileName || '' }
+				isOpen={ showWarningsModal }
+				onClose={ () => setShowWarningsModal( false ) }
+			/>
+
 			{ validationError && (
 				<Notice
 					status="error"
-					isDismissible={ true }
+					isDismissible={ false }
 					onRemove={ () => setValidationError( null ) }
-					className="mx-3 mb-4"
+					className="mx-0 mb-4"
 				>
 					<strong>{ __( 'Blueprint validation failed' ) }</strong>
 					<br />
 					{ validationError }
 				</Notice>
 			) }
 
+			{ ! validationError && blueprintWarnings && blueprintWarnings.length > 0 && (
+				<Notice status="warning" isDismissible={ false } className="mx-0 mb-4">
+					<div className="flex justify-between items-center w-full">
+						<span>
+							{ __(
+								'This blueprint uses some unsupported features in Studio and might not work as expected.'
```

Comment:
```suggestion
								'This blueprint uses unsupported features in Studio and might not work as expected.'
```

Would it sound better without saying some?